website/docs/reference/api/legacy/unleash/admin/features-v2.md

```diff
@@ -378,16 +378,7 @@ Vary: Accept-Encoding
     "name": "DemoNew",
     "project": "default",
     "stale": false,
-    "type": "release",
-    "variants": [
-        {
-            "name": "blue",
-            "overrides": [],
-            "stickiness": "default",
-            "weight": 1000,
-            "weightType": "variable"
-        }
-    ]
```

Comment:
Were they previously included or was the doc wrong here?

Reply:
I went back in time and this was present up until Unleash 4.17. With 4.18 it got removed, probably because variants were no longer at the feature level. Unfortunately, a breaking change.

Reply:
I left it untouched as it's legacy documentation